### ISSUE TITLE
Unable to add route rules if the targets(All Gateways) are in different compartment using OKIT

### ISSUE THREAD

**Issue by KirankBadiger** [oracle/oci-designer-toolkit#452]
1. Unable to add the route rules if the targets(All Gateways) are in different compartment using OKIT

2. Tried to view the existing infrastructure that are deployed in nested compartments even though unable view the route rules

For your reference attached the image below, used OKIT "query" feature to view the existing infrastructure deployed in nested compartments, still unable to view the existing route rules in OKIT

![image](https://user-images.githubusercontent.com/32507539/132375067-8f11b8b3-9df4-440f-8a11-738102e62db7.png)

**Comment by toxophilist** [comment]
@KirankBadiger was the compartment containing the gateway in the query results. Just tried this and can't reproduce if the compartment with the gateway is in the results. If it's not in the result then I get the same empty display but that's not surprising.

**Comment by KirankBadiger** [comment]
@toxophilist gateways are deployed in different compartment, subnets are in different compartments, checked in OCI console, there I can add route rules if the gateways are in different compartment, but with OKIT it is not possible, OKIT should show the existing route rules right?, but it is doesn't

**Comment by toxophilist** [comment]
@KirankBadiger OKIT knows about what is in the design so if all compartments are in the design then is should show the rules correctly. If the compartment is missing from the design then it will not be present for OKIT to display.

**Comment by KirankBadiger** [comment]
@toxophilist  All Compartments are present in OKIT, but route rules are not getting displayed

**Comment by toxophilist** [comment]
@KirankBadiger just spotted your route table is read only which will stop you adding / modifying data

**Comment by KirankBadiger** [comment]
@toxophilist Yes I know it is existing read-only route table, used "Query" feature, but it should show the existing route rules right?

**Comment by toxophilist** [comment]
@KirankBadiger If the Network end point is in the design it should be displayed. Can you check the json view and validate the ocid in the rule looks fine. Just trying to work out where the issue is.
